babel.config.js

```diff
@@ -3,6 +3,6 @@ module.exports = function (api) {
   return {
     presets: ['babel-preset-expo'],
     // Reanimated plugin has to be listed last.
-    plugins: ['inline-dotenv', 'react-native-reanimated/plugin'],
+    plugins: ['inline-dotenv'],
```

Comment:
Pourquoi ce changement ? 
Tu peux retirer le commentaire du dessus aussi

Reply:
cf le ticket ça vient de la note de release du SDK "we now add the Reanimated plugin by default when it's installed (you don't need to remove it from your babel.config.js, but you may want to)."
https://expo.dev/changelog/2024/01-18-sdk-50